nc-news/src/components/Dropdown.jsx

```diff
@@ -1,44 +1,40 @@
-import React, {useEffect, useState} from 'react';
-//import {MenuItems} from './MenuItems'
-import {Link} from 'react-router-dom' 
-import './componentsCSS/Dropdown.css'
-import * as api from '../api'
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import "./componentsCSS/Dropdown.css";
+import * as api from "../api";
 
-function Dropdown(){
-    const [click, setClick] = useState(false)
-    const [categories, setCategory] = useState([])
+function Dropdown() {
+  const [click, setClick] = useState(false);
+  const [categories, setCategory] = useState([]);
 
-    let MenuItems = [
-      {  topic: 'All',
-          path: '/articles',
-          name: 'dropdown-link'
-        }]
+  let MenuItems = [{ topic: "All", path: "/articles", name: "dropdown-link" }];

```

Comment:
this is a bit unusual

You've not set this is state so it won't persist

The way you've set this up means that every time a user mouses-over the `Categories 🔽` drop down it **re-fetches** the topics from the api - on a slow connection this might get quite annoying!

It would be better to fetch once and hold this in state - this might require a bit of a refactor

Reply:
(be sure to use correct camelCase too)

```diff
- MenuItems
+ menuItems
```